tests/Integrations/CLI/NoAutoloader/test-script.php

```diff
@@ -0,0 +1,3 @@
+<?php
```

Comment:
Can we move this to the `tests/Frameworks/Custom/Version_Not_Autoloaded/` folder? :)